master/reference/requirements.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Calico System Requirements
+redirect_from: latest/reference/node/configuration
+---
+
+Depending on the Calico functionality you are using, there are some requirements your system needs to meet in order for Calico to work properly.
+
+### Minumum Linux kernel versions
+IPv4 only: 2.6.32 
+IPv6: 3.10
+
+## Requirements for Calico Policy:
+
+iproute2: iproute2 is a collection of utilities for controlling TCP/IP networking and traffic control in Linux.
+ Kernel modules dependencies: 
+- glibc
+- iptables
+- libelf
+ 
+iputils: The iputils package is set of small useful utilities for Linux networking.
+ Kernel modules dependencies: 
+ - libcap
+ - libidn
+ - openssl
+ - sysfsutils
+ 
+conntrack: The [conntrack-tools](http://www.netfilter.org/projects/conntrack-tools/index.html) are a set of tools to manage the in-kernel connection tracking state table from userspace.
+ Minimum required version: v1.4.1 
+ Kernel modules dependencies: 
+ - libnetfilter_conntrack 
+ - libnfnetlink 
+ - libmnl
+ - libnetfilter_cttimeout
+ - nf_conntrack_netlink subsystem
+    - nf_conntrack
+    - nfnetlink
+ 
+This is included in kernel version 2.6.18 and above.
+ 
+iptables (ip6tables): [iptables](http://www.netfilter.org/projects/iptables/index.html) is a command line utility for configuring Linux kernel firewall implemented within the [Netfilter](http://www.netfilter.org) project.
+ Minimum required version: v1.4.7 
+ Kernel modules dependencies: 
+ - ip_tables (for IPv4)
+    - x_tables
+ - ip6_tables (for IPv6)
+    - x_tables
+ - glibc
+ - libnftnl
+ - libpcap
+ 
+ 
+`x_tables` has the shared code used by iptables modules.
+ This is included in kernel version v2.4 and above.
+ 
+
+ipset: [ipset](http://ipset.netfilter.org/) is used to set up, maintain and inspect so called IP sets in the Linux kernel.
+ Minimum required version: v6.11
+  Kernel modules dependencies: 
+ - ip_set
+    - nfnetlink
+  
+#### Other required kernel modules
+
+xt_set: Kernel module which implements the set match and SET target for netfilter/iptables.
+ - ip_set
+ - x_tables
+ 
+ipt_set: Kernel module to match an IP set.
+ - x_tables
+ - ip_set
+ 
+ 
+ 
+Mark:
+xt_mark
+   - x_tables
+    
+xt_addrtype (ipt_addrtype, ip6t_addrtype) 
+   - x_tables
+
+xt_multiport
+   - x_tables
+ 
+
+## Requirements for Calico Networking:
+ 
+ipip: IP tunnel driver to provide an IP tunnel through which you can tunnel network traffic transparently across subnets.
+ - ip_tunnel
```

Comment:
I think it's unclear from the current formatting what is a module vs what is a feature that uses a module.